Task_TheGridProblem/index.css

```diff
@@ -1 +1,38 @@
-/* Start coding below for GRID PROBLEM TASK */
+/* Start coding below for GRID PROBLEM TASK */

```

Comment:
There seems to be an unnecessary asterisk (*) at the beginning of line 2. It should be removed.